lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1262,51 +1051,55 @@ public function getDateSubYearsExpression($date, $years)
      *
      * @param string $date     The column or literal representing a date to perform the arithmetic operation on.
      * @param string $operator The arithmetic operator (+ or -).
-     * @param int    $interval The interval that shall be calculated into the date.
+     * @param string $interval The interval that shall be calculated into the date.
      * @param string $unit     The unit of the interval that shall be calculated into the date.
-     *                         One of the DATE_INTERVAL_UNIT_* constants.
-     *
-     * @return string
+     *                                 One of the DATE_INTERVAL_UNIT_* constants.
      *
      * @throws DBALException If not supported on this platform.
      */
-    protected function getDateArithmeticIntervalExpression($date, $operator, $interval, $unit)
+    protected function getDateArithmeticIntervalExpression(string $date, string $operator, string $interval, string $unit) : string
     {
         throw DBALException::notSupported(__METHOD__);
     }
 
     /**
-     * Returns the SQL bit AND comparison expression.
+     * Converts the value of a natively unsupported date interval to a value of a supported one
      *
-     * @param string $value1
-     * @param string $value2
+     * @param string $interval   SQL expression describing the value of the interval
+     * @param string $unit       Interval unit
+     * @param int    $multiplier Interval multiplier
      *
-     * @return string
+     * @throws DBALException
      */
-    public function getBitAndComparisonExpression($value1, $value2)
+    protected function convertNonNativeInterval(string $interval, string $unit, int $multiplier) : string
```

Comment:
`$multiplier` should be `int` according to phpDoc so this is a BC break. Also visibility change is a BC break.

Reply:
There's no visibility change. It's a different method.